code/datums/ai_law_sets.dm

```diff
@@ -25,7 +25,7 @@
 
 /******************** Nanotrasen/Malf ********************/
 /datum/ai_laws/nanotrasen
-	name = "NT Default"
+	name = "NT Recommended"
```

Comment:
Less of a code comment, more of a... I really don't like this name change... You can't call something 'recommended' then not have it come as a default.